### DISCUSSION TITLE
⚙️What are the available arguments I can specify to modify how SystemInformer executes?

### DISCUSSION THREAD

**Discussion by SalmonSnakeD** [winsiderss/systeminformer#2160] [Questions / Answers] [answered]
I'm trying to find a way to start SystemInformer with elevated permissions and real-time priority.
It used to be like this `-elevate -priority r` **with ProcessHacker**, but I couldn't find any docs on the arguments SystemInformer uses.

**Comment by dmex** [comment, marked as answer]
The website is getting updated with documentation such as the commands.

The `-elevate` and `priority` options are still the same. Here's a printout from the legacy documentation:

[commands.pdf](https://github.com/user-attachments/files/16652425/commands.pdf)
